code/Commenting.php

```diff
@@ -13,12 +13,18 @@
  */
 
 class Commenting {
-	
+
 	/**
 	 * @var array map of enabled {@link DataObject} and related configuration
 	 */
 	private static $enabled_classes = array();
-	
+
+	/**
+	 * @config
+	 * @var bool Whether to enable commenting on SiteTree objects by default
+	 */
+	private static $sitetree_comments = true;
+
```

Comment:
How about we just remove it completely from _config.php and let people manually set it up?